Userland/Applications/GamesSettings/CardSettingsWidget.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022, Sam Atkins <atkinssj@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "CardSettingsWidget.h"
+#include <Applications/GamesSettings/CardSettingsWidgetGML.h>
+#include <LibCards/CardPainter.h>
+#include <LibConfig/Client.h>
+#include <LibGUI/FileSystemModel.h>
+#include <LibGfx/Palette.h>
+
+static constexpr StringView default_card_back_image_path = "/res/icons/cards/buggie-deck.png"sv;
+
+CardSettingsWidget::CardSettingsWidget()
+{
+    load_from_gml(card_settings_widget_gml);
+
+    auto background_color = Gfx::Color::from_string(Config::read_string("Games"sv, "Cards"sv, "BackgroundColor"sv)).value_or(Gfx::Color::from_rgb(0x008000));
+
+    m_preview_frame = find_descendant_of_type_named<GUI::Frame>("cards_preview");
+    set_cards_background_color(background_color);
+
+    m_preview_card_back = find_descendant_of_type_named<GUI::ImageWidget>("cards_preview_card_back");
+    m_preview_card_back->set_bitmap(Cards::CardPainter::the().card_back());
+
+    m_preview_card_front_ace = find_descendant_of_type_named<GUI::ImageWidget>("cards_preview_card_front_ace");
+    m_preview_card_front_ace->set_bitmap(Cards::CardPainter::the().card_front(Cards::Suit::Spades, Cards::Rank::Ace));
+    m_preview_card_front_queen = find_descendant_of_type_named<GUI::ImageWidget>("cards_preview_card_front_queen");
+    m_preview_card_front_queen->set_bitmap(Cards::CardPainter::the().card_front(Cards::Suit::Hearts, Cards::Rank::Queen));
+
+    m_background_color_input = find_descendant_of_type_named<GUI::ColorInput>("cards_background_color");
```

Comment:
let’s maybe disable the alpha channel in the color picker? It doesn’t work anyway

```suggestion
    m_background_color_input = find_descendant_of_type_named<GUI::ColorInput>("cards_background_color");
    m_background_color_input->set_color_has_alpha_channel(false);
```

Reply:
Thanks! I didn't realise that was an option. I've done that in the GML. :^)